query_delete.php

```diff
@@ -24,6 +24,8 @@
  * @copyright Copyright 2002  MantisBT Team - mantisbt-dev@lists.sourceforge.net
  * @link http://www.mantisbt.org
  *
+ * @deprecated This page has been replaced by 'manage_filter_delete.php'
```

Comment:
What is the reason for holding on to the deprecated pages rather than deleting them?  Sorry, if we discussed this before.